cicecore/cicedynB/analysis/ice_history.F90

```diff
@@ -538,22 +568,32 @@ subroutine init_hist (dt)
              "snow/ice surface temperature",                      &
              "averaged with Tf if no ice is present", c1, c0,     &
              ns1, f_Tsfc)
-      
+
          call define_hist_field(n_aice,"aice","1",tstr2D, tcstr,    &
              "ice area  (aggregate)",                             &
              "none", c1, c0,                                      &
              ns1, f_aice)
       
-         call define_hist_field(n_uvel,"uvel","m/s",ustr2D, ucstr,  &
+         call define_hist_field(n_uvel,"uvel","m/s",l_ustr2D, l_ucstr,  &
              "ice velocity (x)",                                  &
-             "positive is x direction on U grid", c1, c0,         &
+             "positive is x direction on u grid", c1, c0,         &
              ns1, f_uvel)
       
-         call define_hist_field(n_vvel,"vvel","m/s",ustr2D, ucstr,  &
+         call define_hist_field(n_vvel,"vvel","m/s",l_vstr2D, l_vcstr,  &
              "ice velocity (y)",                                  &
-             "positive is y direction on U grid", c1, c0,         &
+             "positive is y direction on v grid", c1, c0,         &
```

Comment:
Correct.  The U grid is now defined at the corner point.  The u and v grid could be the U, N, or E grid.  We do need to work on making this clearer in history output, but we do write the history coordinates as part of the output as well, so that helps.

Reply:
I'd like this one fixed for this PR, please.  It's confusing (to me) otherwise.  BTW did you spot check the history output from a run to make sure it's working as expected, including existing variables like uvel?

Reply:
OK, I'll refactor it a bit.  I have checked that I can vary the underlying coordinate system for history fields based on grid_subsystem.  One other thing, these comments that are created for variables are not actually written to the history file right now.  I'm not sure why, they certainly could be.  But it's not implemented in the netcdf or pio IO versions.

Reply:
Weird that they aren't written -- they definitely should be!  Another thing to follow up on...

Reply:
I have updated the history writing and comments are now being written.